tests/tests.js

```diff
@@ -652,6 +652,22 @@ suite('Form Element Bindings', function() {
     radioInputCheckedMultipleForms(shadowRoot, done);
   });
 
+  test('(Number)Input.number', function(done) {
```

Comment:
given the other `(Radio)Input.checked` test, maybe this should be `(Number)Input.value` for consistency?